.coderabbit.yaml

```
@@ -0,0 +1,57 @@
+# Language settings
+language: ja
+
+# Review settings
+reviews:
+  # Enable automatic reviews
+  auto_review:
+    enabled: true
+    
+  # Review level settings
+  review_status: "all_files"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)